src/map.rs

```diff
@@ -2087,18 +2061,17 @@ where
             // there are three cases when a bin pointer is invalidated:
             //
             //  1. if the table was resized, bin is a move entry, and the resize has completed. in
-            //     that case, the table (and all its heads) will be dropped in the next epoch
-            //     following that.
+            //     that case, the table (and all its heads) will be retired.
```

Comment:
same notes as for the earlier instance of this argument

Reply:
Fixed.